test/tests/functional/pbs_maintenance_reservations.py

```diff
@@ -440,36 +440,53 @@ def test_maintenance_and_reservation_not_overlap2(self):
                     'reserve_substate': 2}
         self.server.expect(RESV, exp_attr, id=rid1)
 
+    def get_reg_expr(self, _mom, _host):
+        """
+        This function forms the match string based on cpuset mom
+        """
+        if _mom.is_cpuset_mom():
```

Comment:
Since this function is creating regex strings, do you want to use r'str' for raw strings?

Reply:
I tried using r'str' but expect function is searching with str(reg_expr).
elif op == MATCH_RE:
    if re.search(str(v), str(stat_v)):
        continue
My reg_str string remains same even if I use r'str' except checking for ncpus number. I have to include backslashes for every special character.

Reply:
OK, that's fine.  The other option is to use re.escape() to handle the backslashes for you.  What you are doing is fine though.